core/math/big/prime.odin

```diff
@@ -112,6 +112,9 @@ internal_int_exponent_mod :: proc(res, G, X, P: ^Int, allocator := context.alloc
 	*/
 	return _private_int_exponent_mod(res, G, X, P, 0)
 }
+internal_int_exponent_mod :: internal_int_exponent_modulus
+internal_int_powmod :: internal_int_exponent_modulus
+internal_powmod :: proc { internal_int_powmod, }
```

Comment:
Same here with the proc group. Just point it to the actual thing:
`internal_powmod :: proc{ internal_int_exponent_modulo, }`

And `_modulo` instead of modulus, because see above.